UI/window-projector.cpp

```diff
@@ -11,14 +11,39 @@
 #include "platform.hpp"
 #include "multiview.hpp"
 
+#ifdef ENABLE_WAYLAND
+#include <obs-nix-platform.h>
+#endif
+
+#ifdef _WIN32
+#define NOMINMAX
+#define WIN32_LEAN_AND_MEAN 1
+#include <Windows.h>
+#endif
+
 static QList<OBSProjector *> multiviewProjectors;
 
 static bool updatingMultiview = false, mouseSwitching, transitionOnDoubleClick;
 
+static bool IsWayland()
+{
+#ifdef ENABLE_WAYLAND
+	return obs_get_nix_platform() == OBS_NIX_PLATFORM_WAYLAND;
+#else
+	return false;
+#endif
+}
```

Comment:
Prefer checking Qt's backend than libobs for this case.

```
QApplication::platformName().contains("wayland")
```

Reply:
Fixed